lib/util/git/index.ts

```diff
@@ -850,7 +850,7 @@ export async function prepareCommit({
         if (await isDirectory(upath.join(localDir, fileName))) {
           // This is usually a git submodule update
           logger.trace({ fileName }, 'Adding directory commit');
-        } else if (file.contents === null) {
+        } else if (is.falsy(file.contents)) {
```

Comment:
Not sure about this change. Will modify if this isn't okay.

Reply:
this is wrong, empty string is allowed